secretMP/main.py

```diff
@@ -1,12 +1,14 @@
 import stt
+#from stt import listen_for_wake_word
 import tts
 from tts import convert_to_good_audio, get_gpt4_response
+import time
 
 def main():
-    spoken_text = stt.capture_and_transcribe() # transcribe the text (working well)
-    #save_this = tts.get_gpt4_response(spoken_text) # take the transcribed text and pass it to gpt
-    #msave_text_output = tts.get_gpt4_response(save_this)
-    tts.convert_to_good_audio(spoken_text)
+    while True: 
+        #stt.listen_for_wake_word()
```

Comment:
We might want to add a waitKey() so users can click q on the JS application to stop the while loop.